workfinder/workfinder.py

```diff
@@ -7,19 +7,24 @@
 
 # Mapping of processor names to objects.
 # Note: Names must be upper case.
+from workfinder.search.s1_ard import S1ARD
+
 processors = {
-    "S1": S1()
+    "S1": S1(),
+    "S1_ARD": S1ARD()
 }
 
 
 def main():
     if len(sys.argv) > 1:
         param = sys.argv[1]
+        # TODO: Limit parameter
```

Comment:
I recommend you the use of [click library](https://click.palletsprojects.com/en/8.0.x/), it is really good for CLI tools

Reply:
Looks like a good choice, I'll switch it over in a later pr as I've completed this todo comment so its going to need unpicking twice if I do it here.